vulnerabilities/tests/test_api.py

```diff
@@ -194,95 +194,73 @@ def test_package_serializer(self):
+        ).json()
+
+        expected_response = [
+            {},
+            {
+                "url": "http://testserver/api/packages/3467",
                 "unresolved_vulnerabilities": [
-                    OrderedDict(
-                        [
-                            ("url", "http://testserver/api/vulnerabilities/1"),
-                            ("vulnerability_id", "CVE-2014-8242"),
-                        ]
-                    )
-                ],
-            },
-            "pkg:deb/debian/mimetex@1.50-1.1?distro=jessie": {
-                "resolved_vulnerabilities": [
-                    OrderedDict(
-                        [
-                            ("url", "http://testserver/api/vulnerabilities/2"),
-                            ("vulnerability_id", "CVE-2009-1382"),
-                        ]
-                    ),
-                    OrderedDict(
-                        [
-                            ("url", "http://testserver/api/vulnerabilities/3"),
-                            ("vulnerability_id", "CVE-2009-2459"),
-                        ]
-                    ),
+                    {
+                        "url": "http://testserver/api/vulnerabilities/60",
+                        "vulnerability_id": "CVE-2021-21331",
+                        "references": [
+                            {
+                                "source": "",
+                                "reference_id": "GHSA-2cxf-6567-7pp6",
+                                "url": "https://github.com/advisories/GHSA-2cxf-6567-7pp6",
+                                "scores": [],
+                            },
+                            {
+                                "source": "",
+                                "reference_id": "",
+                                "url": "https://nvd.nist.gov/vuln/detail/CVE-2021-21331",
+                                "scores": [],
+                            },
+                            {
+                                "source": "",
+                                "reference_id": "GHSA-2cxf-6567-7pp6",
+                                "url": "https://github.com/DataDog/datadog-api-client-java/security/advisories/GHSA-2cxf-6567-7pp6",
+                                "scores": [{"value": "LOW", "scoring_system": "cvssv3.1_qr"}],
+                            },
+                        ],
+                    }
                 ],
-                "unresolved_vulnerabilities": [],
+                "resolved_vulnerabilities": [],
+                "purl": "pkg:maven/com.datadoghq/datadog-api-client@1.0.0-beta.7",
+                "type": "maven",
+                "namespace": "com.datadoghq",
+                "name": "datadog-api-client",
+                "version": "1.0.0-beta.7",
+                "subpath": "",
+                "qualifiers": {},
             },
-        }
+        ]
 
-        assert response == expected_response
+        # This normalization is brittle
```

Comment:
Use something less ugly that does the clean up in a clear way., e.g. more or less..
```suggestion
def clean_response(response):
    """
    Return a cleaned response suitable for comparison in tests in particular:
    - sort lists with a stable order
    - remove transient values such as primary keys or dates that may change between calls.
    """
    for item in response:
        for unresolved_vulnerability in item["unresolved_vulnerabilities"]:
                .....         
        
    
                # This normalization is brittle
```